{
  final Class<?> testClass=prepareTest();
  try {
    testClass.getMethod("main").invoke(null);
    fail();
  }
 catch (  InvocationTargetException e) {
    assertInstanceOf(e.getCause(),IllegalArgumentException.class);
    String trace=ExceptionUtil.getThrowableText(e.getCause());
    assertEquals("Exception should happen in real, non-bridge method: " + trace,2,StringUtil.getOccurrenceCount(trace,"B.getObject(SkipBridgeMethods"));
  }
}
